stores/sql/main.go

```diff
@@ -2229,8 +2236,8 @@ func Settings(ctx context.Context, tx sql.Tx) ([]string, error) {
 	return settings, nil
 }
 
-func SetUncleanShutdown(ctx context.Context, tx sql.Tx) error {
-	_, err := tx.Exec(ctx, "UPDATE ephemeral_accounts SET clean_shutdown = 0, requires_sync = 1")
+func SetUncleanShutdown(ctx context.Context, tx sql.Tx, owner string) error {
```

Comment:
I'd make `owner` optional here as well

Reply:
This is actually not used anymore anyway